.github/actions/setup-rubygems.org/action.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Setup rubygems.org'
```

Comment:
does extracting this into an action actually help make things faster?

Reply:
No, it doesn't not. This change is not directly related, but makes it a little easier to maintain workflows. My original plan was to use multiple workflows reusing this shared action, but in the end I found out it is still simple enough to use matrix within one workflow job. For now I can revert this change and keep it locally unless it will be really beneficial.

Reply:
nah splitting it out is fine, just wanted to understand